internal/pkg/describe/lb_web_service.go

```diff
@@ -189,34 +208,47 @@ func (d *LBWebServiceDescriber) Describe() (HumanJSONStringer, error) {
 	}, nil
 }
 
-// URI returns the LBWebServiceURI to identify this service uniquely given an environment name.
+// URI returns the WebServiceURI to identify this service uniquely given an environment name.
```

Comment:
nit: this comment says "WebServiceURI" but it's from a "LBWebServiceURI" can we rename the type?